assets/blueprints/blueprint.json

```diff
@@ -0,0 +1,49 @@
+      },
+      {
+        "step": "installPlugin",
+        "pluginZipFile": {
+          "resource": "wordpress.org/plugins",
+          "slug": "glotpress"
+        }
+      },
+      {
+        "step": "runPHP",
+        "code": "<?php include 'wordpress/wp-load.php';// Ensure the database tables are created.require_once ABSPATH . 'wp-admin/includes/upgrade.php';require_once GP_PATH . GP_INC . 'install-upgrade.php';require_once GP_PATH . GP_INC . 'schema.php';gp_upgrade_db();set_current_user( 1 );$request = new WP_REST_Request();$request->set_param( 'name', 'friends');$request->set_param( 'path', 'wp-' . 'plugin' . 's/' . 'friends' );$request->set_param( 'locale', 'de_DE' );$request->set_param( 'locale_slug', 'default' );print_r( GP::$rest->create_local_project( $request ) );$glossary_entry_route = new GP_Route_Glossary_Entry();if ( file_exists( '/wordpress/wp-content/languages/glossary-de_DE.csv' ) ) {$locale_glossary_translation_set = GP::$translation_set->by_project_id_slug_and_locale( 0, 'default', 'de' );$locale_glossary = new GP_Glossary();$locale_glossary->translation_set_id = $locale_glossary_translation_set->id;$locale_glossary = GP::$glossary->create_and_select( $locale_glossary );echo $glossary_entry_route->read_glossary_entries_from_file( '/wordpress/wp-content/languages/glossary-de_DE.csv', $locale_glossary->id, 'de' ), ' locale glossary entries', PHP_EOL;}if ( file_exists( '/wordpress/wp-content/languages/plugins/friends-glossary-de_DE.csv' ) ) {$project = GP::$project->by_path( 'local-plugins/friends' );$glossary_translation_set = GP::$translation_set->by_project_id_slug_and_locale( $project->id, 'default', 'de' );$glossary = new GP_Glossary();$glossary->translation_set_id = $glossary_translation_set->id;$glossary = GP::$glossary->create_and_select( $glossary );echo $glossary_entry_route->read_glossary_entries_from_file( '/wordpress/wp-content/languages/plugins/friends-glossary-de_DE.csv', $glossary->id, 'de' ), ' project glossary entries', PHP_EOL;;}",
```

Comment:
I don't think we need the glossaries for this live preview, they are just necessary for translate live.